src/model/AirLoopHVACOutdoorAirSystem.cpp

```diff
@@ -204,14 +220,47 @@ namespace model {
       // Clone relief stream comps
 
       auto reliefComps = reliefComponents();
+
+      auto reliefAirToAirComps = subsetCastVector<AirToAirComponent>(reliefComps);
+
       std::vector<Node> reliefNodes;
 
-      for (const auto& comp : reliefComps) {
-        if (comp.iddObjectType() == Node::iddObjectType()) {
-          reliefNodes.push_back(comp.cast<Node>());
-        } else {
-          auto compClone = comp.clone(model).cast<HVACComponent>();
-          compClone.addToNode(reliefNodeClone);
+      if (reliefAirToAirComps.empty()) {
+        // Business as usual, but this is faster this way as we keep the same node to connect to throughout
+        std::vector<Node> reliefNodes;
+
+        for (const auto& comp : reliefComps) {
+          if (comp.iddObjectType() == Node::iddObjectType()) {
+            reliefNodes.push_back(comp.cast<Node>());
+          } else {
+            auto compClone = comp.clone(model).cast<HVACComponent>();
+            compClone.addToNode(reliefNodeClone);
+          }
+        }
+      } else {
+        auto targetNode = oaclone.reliefAirModelObject()->cast<Node>();
+
+        for (const auto& comp : reliefComps) {
+          if (comp.iddObjectType() == Node::iddObjectType()) {
+            reliefNodes.push_back(comp.cast<Node>());
+          } else if (comp.optionalCast<AirToAirComponent>()) {
+            auto reliefCloneMo_ = targetNode.outletModelObject();
+            OS_ASSERT(reliefCloneMo_);
+            auto reliefCloneAirToAir_ = reliefCloneMo_->optionalCast<AirToAirComponent>();
+            OS_ASSERT(reliefCloneAirToAir_);
+            auto mo_ = reliefCloneAirToAir_->secondaryAirOutletModelObject();
+            OS_ASSERT(mo_);
+            targetNode = mo_->cast<Node>();
+          } else {
+            auto compClone = comp.clone(model).cast<HVACComponent>();
+            compClone.addToNode(targetNode);
+            auto [ok, mo_] = getOutletNodeForComponent(compClone);
+            if (!ok) {
+              LOG_AND_THROW("For " << briefDescription() << ", unexpected reliefComponent found: " << compClone.briefDescription());
+            }
+            OS_ASSERT(mo_);
+            targetNode = mo_->cast<Node>();
+          }
```

Comment:
Actual fix is here.... When we have an ERV, we need to be smarter on the relief side, because cloning the oa intake side already added the ERVs. So we track the outlet nodes instead of trying to add to the relief air node constantly.

Reply:
I looked through all of this PR and thought ok this is great, lots of good cleanup and small fixes, but where is the actual fix to the bug? Then I realized github had collapsed the diff in this file because it was a large diff. So here we go. I appreciate having the fast path if there are no ERVs.